src/test/java/nextstep/member/acceptance/AuthAcceptanceTest.java

```diff
@@ -36,4 +41,43 @@ void bearerAuth() {
         assertThat(memberInfoResponse.statusCode()).isEqualTo(HttpStatus.OK.value());
         assertThat(memberInfoResponse.jsonPath().getString("email")).isEqualTo(EMAIL);
     }
+
+    /**
+     *  Given : 가입한 회원이 존재하고
+     *  When : 해당 회원이 깃헙 로그인을 요청하면
+     *  Then : 토큰을 발급한다.
+     */
+    @Test
+    void 회원이_깃헙로그인을_한다() {
+        // given
+        회원_생성_요청(사용자1.getEmail(), "password", 20);
+        var request = new GithubTokenRequest(사용자1.getCode());
+
+        // when
+        ExtractableResponse<Response> response = 깃헙_로그인_요청(request);
```

Comment:
response의 타입이 길다보니 가독성을 해친다고 느껴질 수 있는데요~
var를 사용하면 인수 테스트 작성 시 조금 더 가독성이 살아날 수 있습니다 :) 

```suggestion
        var response = 깃헙_로그인_요청(request);
```